tests/framework/base/SecurityTest.php

```diff
@@ -5,6 +5,18 @@
  * @license http://www.yiiframework.com/license/
  */
 
+namespace yii\base;
+
+function function_exists($name) {
```

Comment:
Would be nice to have an explanation comment here. It is hard to understand the meaning of this trick at the first glance.